master/internal/api_master.go

```diff
@@ -110,17 +109,11 @@ func (a *apiServer) PatchMasterConfig(
 	for _, path := range paths {
 		switch path {
 		case "log":
-			if !isValidLogLevel(req.Config.Log.Level) {
-				panic(fmt.Sprintf("invalid log level: %s", req.Config.Log.Level))
-			}
-			a.m.config.Log.Level = req.Config.Log.Level
+			a.m.config.Log.Level = req.Config.Log.Level.String()[10:]
```

Comment:
I don't think this set of `a.m.config.Log.Level` is threadsafe.

Reply:
I will address this in a separate PR